main.py

```diff
@@ -23,7 +24,7 @@ def get(self,permPath):
         cn = sqlite3.connect('makermon.db')
         c = cn.cursor()
         fob_id = paths[1]
-        sql = 'SELECT * FROM members WHERE fob_id = "%s"' % ( fob_id )
+        sql = 'SELECT * FROM members m, rfid_tokens r WHERE m.enabled = 1 AND r.member_id = m.id AND r.enabled = 1 AND r.token = "%s"' % ( fob_id )
```

Comment:
My only problem with this is that it leaves the door open for sql injection. Imagine if fob_id came in as something hostile (like `1"; DROP TABLE members; --`).  
  
Take a look at [the execute() method which takes two params](https://docs.python.org/2/library/sqlite3.html). That should solve it.

Reply:
That makes sense.  I was wondering what the two different syntax was for, I just had not looked it up yet.  I knew SQL injection was a concern, didn't know that paramatizing the execute call took care of it automatically.

How do I fix this in a pull request on GitHub.  Do I have to cancel the pull request update the branch and then make another pull request?  Or is there some way to update the pull request automatically when updating the branch?

Reply:
It's the latter - when you push your next commit to this branch, the pull request will be updated with the latest and greatest.